apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	);
 
 	return (
-		<section className={styles["signup-container"]}>
-			<div className={styles["logo-container"]}>
-				<div
-					className={getValidClassNames(
-						styles["circle"],
-						styles["circle-pink"],
-					)}
-				/>
-				<h1>LOGO</h1>
-			</div>
-			<h3 className={styles["title-form"]}>CREATE AN ACCOUNT</h3>
-			<span className={styles["auth-info"]}>
-				Already have an account? Go to
-				<Link to={AppRoute.SIGN_IN}> Sign in</Link>
-			</span>
-			<form onSubmit={handleFormSubmit}>
-				<div className={styles["input-groups"]}>
+		<div className={styles["container"]}>
+			<div className={styles["form-container"]}>
+				<form className={styles["form"]} onSubmit={handleFormSubmit}>
+					<div className={styles["form-header"]}>
+						<div className={styles["form-header__logo-container"]}>
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>logo</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>create an account</h1>
+						<span className={styles["form-header__sub-text"]}>
+							Already have an account? Go to{" "}
+							<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+						</span>
+					</div>
```

Comment:
These are not parts of a form, should be outside of `<form>` tag
Should also be fixed for sign-in screen